test/ruby/test_yjit.rb

```diff
@@ -1461,8 +1463,15 @@ def collect_insns(iseq)
       if exits != :any &&
         exits != recorded_exits &&
         (exits.keys != recorded_exits.keys || !exits.all? { |k, v| v === recorded_exits[k] }) # triple-equal checks range membership or integer equality
-        flunk "Expected #{exits.empty? ? "no" : exits.inspect} exits" \
-          ", but got\n#{recorded_exits.inspect}"
+        stats_reasons = StringIO.new
+        ::RubyVM::YJIT.send(:_print_stats_reasons, runtime_stats, stats_reasons)
+        stats_reasons = stats_reasons.string
+        flunk <<~EOM
+          Expected #{exits.empty? ? "no" : exits.inspect} exits ", but got
+          #{recorded_exits.inspect}"
```

Comment:
```suggestion
          Expected #{exits.empty? ? "no" : exits.inspect} exits, but got
          #{recorded_exits.inspect}
```

Are these quotes necessary? 🤔